java/java.api.common/build.xml

```diff
@@ -21,5 +21,21 @@
 -->
 <project basedir="." default="build" name="java/java.api.common">
     <import file="../../nbbuild/templates/projectized.xml"/>
+
+    <target name="generate-antlr-parser" description="Regenerate the CSS parser from the ANTLR3 grammar">
```

Comment:
Do we want to add a dependency with the compile task like so?
```
<target name="compile" depends="generate-antlr-parser">
</target>
```

Reply:
Thank you for the review. I actually thought about this (and planned to remove the generated java files from the repository), but the generated code is not reproducible.
Antlr3 injects the generation time and paths into the files and at least from a quick look, this could not be overriden. Debian carries a patch the also changes the source, so that reproducible datastructures are used (https://sources.debian.org/patches/antlr3/3.5.2-9/05-reproducible-parsers.patch/). 
So the code will change on each build and from my perspective this does not scale. In this case IMHO it is better to keep the generated parsers and regenerate them when needed.

Reply:
I see. Good to avoid chaning the code on each build then.